english/stapler/no.md

```diff
@@ -0,0 +1 @@
+![Burning Building](https://media.giphy.com/media/7XqRXKYV9odtm/giphy.gif)
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.